sphinxcontrib/confluencebuilder/translator/wiki.py

```diff
@@ -26,10 +26,10 @@
 
 # Confluence encoding for special chars
 SPECIAL_VALUE_REPLACEMENTS = {
-    ('{', '&lcub;'),
-    ('}', '&rcub;'),
-    ('<', '&lt;'),
-    ('>', '&gt;')
+    ('{', '&#123;'),
```

Comment:
why was this changed? I remember having issues with code points.

Reply:
Interesting. Before the change (from commit b6bb29230db22ccc45a5852c75f88ef4cfbea422), I got an error when using the curly brackets:
```
MSG: Error parsing xhtml: Undeclared general entity "lcub"
 at [row,col {unknown-source}]: [1,390]
```
I'll go back and try to re-produce the issue. I also wonder if any of this relates to the use of the `HTMLParser` (proposed for removal in 255043422d25f8e437baf3182eafa6464a1ab288).

Reply:
Do you have any hints on your observed issues using decimal values? After trying to toggle the change, I always see an error generated when using the '\&lcub;' (and respective) values:

```
An unsupported Confluence API call has been made.

REQ: PUT
RSP: 400
URL: https://jdknight.atlassian.net/wiki/rest/api/
API: content
MSG: Error parsing xhtml: Undeclared general entity "lcub"
 at [row,col {unknown-source}]: [3,30]
```

Always observing errors in 2.7, 3.4-3.6 on both Confluence Server and Confluence Cloud when using '\&lcub;'. No issues with the decimal variant. Thoughts?